### PR TITLE
Disable acceleration Caps

### PR DESCRIPTION
**Pull request by NikhilSuresh24**
switch that enables or disables all acceleration caps

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -12,5 +12,7 @@
 	</classpathentry>
 	<classpathentry kind="var" path="opencv" sourcepath="opencv.sources"/>
 	<classpathentry kind="var" path="cscore" sourcepath="cscore.sources"/>
+	<classpathentry kind="var" path="USERLIBS_DIR/CTRLib-javadoc.jar"/>
```
Comment (c0d3rman) [inline .classpath]: No please
Please cd to the root of your repo and run
```
git checkout master -- .classpath
git add .classpath
git commit
git push
```
The first command gets the version of .classpath master has

**Review by c0d3rman — CHANGES_REQUESTED** [review]

File: src/org/usfirst/frc4904/robot/RobotMap.java [hunk 2]
```diff
@@ -117,6 +119,8 @@
 		public static Climber climber;
 		public static MotionController chassisTurnMC;
 		public static AligningCamera alignCamera;
+		public static EnableableModifier rightWheelAccelerationCap;
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/RobotMap.java]: I don't love having non-material things here, but I can't think of a better place to keep these.

File: src/org/usfirst/frc4904/robot/RobotMap.java [hunk 3]
```diff
@@ -162,7 +168,7 @@ public RobotMap() {
 		// Human inputs
 		Component.operatorStick = new CustomJoystick(Port.HumanInput.joystick);
 		Component.operatorStick.setDeadzone(HumanInterfaceConfig.JOYSTICK_DEADZONE);
-		Component.teensyStick = new CustomJoystick(Port.HumanInput.teensyStick);
+		Component.teensyStick = new TeensyController(Port.HumanInput.teensyStick, 30);
```
Comment (ajnadel) [inline src/org/usfirst/frc4904/robot/RobotMap.java]: Are there actually 30 buttons? Also, can you move the number to a constant, maybe HumanInterfaceConfig.

**Review by ajnadel — APPROVED** [review]

File: src/org/usfirst/frc4904/robot/RobotMap.java (hunk 2 above)
Comment (NikhilSuresh24) [reply src/org/usfirst/frc4904/robot/RobotMap.java]: cool

**Review by ajnadel — APPROVED** [review]

**Review by c0d3rman — APPROVED** [review]

File: src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java [hunk 4]
```diff
@@ -64,7 +64,12 @@ public void bindCommands() {
 		RobotMap.Component.driverXbox.dPad.right.whenReleased(normalDrive);
 		RobotMap.Component.driverXbox.b.onlyWhileHeld(HumanInterfaceConfig.gearAlign);
 		RobotMap.Component.driverXbox.b.whenReleased(normalDrive);
-		RobotMap.Component.teensyStick.button1.whenPressed(normalDrive);
+		RobotMap.Component.teensyStick.getButton(0).whenPressed(normalDrive);
```
Comment (widdakay) [inline src/org/usfirst/frc4904/robot/humaninterface/drivers/NathanGain.java]: Same as what aj said for this, and the rest of the constants for what the teensy uses

**Pull request merged**